.github/workflows/ci.yml

```diff
@@ -104,10 +104,10 @@ jobs:
 
       # Compile / Test / Package are separate steps so the reason for any failure is more obvious in GitHub UI
       - name: Compile
-        run: mvn --batch-mode compile
+        run: mvn --batch-mode compile -Dmaven.compiler.release=${{ matrix.javac_release }}
```

Comment:
I'm not sure if we gain anything building with `maven.compiler.release=11` and `maven.compiler.release=17` because it also builds with `maven.compiler.release=8` so the higher numbers must be ok ?